emperor/support_files/js/view.js

```diff
@@ -395,6 +556,9 @@ DecompositionView.prototype.updatePositions = function() {
     });
     cloud.geometry.attributes.position.needsUpdate = true;
   }
+  else if (this.decomp.isScatterType() && this.viewType === 'parallel-plot') {
+    //TODO:  Do we need to do anything when axes are changed in parallel plots?
```

Comment:
No, in fact this shouldn't happen at all.

Reply:
We hit this line as a result of the controller save to json/load from json hack when switching between scatter and parallel plot.  I do agree that doing nothing here is a bit of a bandaid, but we need to add change tracking (like the UIState properties) to all the saved/loaded properties in order to avoid this.  

Stack that hits this line:
console.trace() view.js:556:13
    updatePositions file:///Users/Dan/knightLab/emperor/emperor/support_files//js/view.js:556
    changeVisibleDimensions file:///Users/Dan/knightLab/emperor/emperor/support_files//js/view.js:689
    fromJSON file:///Users/Dan/knightLab/emperor/emperor/support_files//js/axes-controller.js:760
    forceRefresh file:///Users/Dan/knightLab/emperor/emperor/support_files//js/abc-view-controller.js:192
    render file:///Users/Dan/knightLab/emperor/emperor/support_files//js/controller.js:506
    each jQuery
    render file:///Users/Dan/knightLab/emperor/emperor/support_files//js/controller.js:500
    animate file:///Users/Dan/knightLab/emperor/new-emperor.html:174

Reply:
Got it, this seems fine then.